### PR TITLE
Milestone 2: add reducers and actions

### PR DESCRIPTION
**Pull request by dqarias**
In this milestone we:

- write book's actions and reducer (in `/src/redux/books/books.js`) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).
- write your categories actions and reducer (in `/src/redux/categories/categories.js`) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).
- configure the Redux Store `(/src/redux/configureStore.js`):

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
 import React from 'react';
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import Books from './components/Books';
 import NavBar from './components/Navbar';
 import Categories from './components/Categories';
 import NoMatch from './screens/NoMatch';
+import store from './redux/configureStore';
 import 'App.css';
```
Comment (zieeco) [inline src/App.js]: - Your app is throwing an error because of this import statement, kindly fix it:
```
import './App.css';
```

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @<STUDENT NAME>,


Great work so far! 👏 
You've done well, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (dqarias) [reply src/App.js]: Thanks, I didn't realize about that

**Review by DeliceLydia — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**